# Experimental challenge of African green monkeys with contemporary Hendra virus isolates produces divergent clinical disease: Nanostring transcriptomic analyses

[![DOI](https://zenodo.org/badge/968323978.svg)](https://doi.org/10.5281/zenodo.15238546)

> Paper citation TBD

## Abstract
Hendra virus (HeV) is a medically important, zoonotic paramyxovirus that emerged over thirty years ago which causes severe, often fatal disease in humans and animals. There are presently no medical countermeasures approved to prevent or treat human HeV disease, although many are in various stages of development. Critical to the stringent evaluation of these experimental countermeasures are nonhuman primate models of HeV disease which accurately recapitulate the pathogenesis of human infection. The continued emergence of HeV since its initial discovery in 1994 has recently expanded to include the identification of a new variant with enough genetic divergence to require the creation of a second genotype. Although this variant HeV has produced fatal equine disease, its pathogenesis and lethality are unknown in humans. In the present study, we investigated the pathogenicity of clinically relevant and contemporary HeV isolates from genotype 1 (HeV/Australia/Horse/2008/Redlands) and genotype 2 (HeV-var/Australia/Horse/2015/Gympie) in the African green monkey (AGM) model of henipavirus disease. Experimental challenge of AGMs with 5.0E5 PFU of HeV genotype 1 (HeV-g1) or genotype 2 (HeV-g2) isolates via the combined intranasal/intratracheal route of exposure produced divergent survival outcomes, with four of five AGMs infected with the HeV-g2 isolate surviving until the predetermined scientific endpoint of the study. All five HeV-g1 infected subjects developed acute HeV disease which accurately recapitulated HeV pathogenesis reported in humans. Our findings revealed that HeV-g2 is less pathogenic than HeV-g1 in the AGM animal model and suggests that HeV-g2 may be less pathogenic in humans.

## Methods

Whole blood samples containing HeV-R or HeV-g2 were inactivated in TRIzol LS (Invitrogen) or AVL buffer (Qiagen), and RNA was extracted following removal from BSL-4 containment. RNA isolation was performed according to manufacturer’s recommendation. Extracted RNA from TRIzol LS-inactivated samples was concentrated and DNAse digested with RNA Clean and Concentration-5 kit (Zymo Research). Purified RNA samples were hybridized with the NHP_Immunology_V2 reporter and capture codesets containing ~779 targets (Nanostring Technologies) for ~24h at 65˚C and subsequently loaded into nCounter microfluidics cartridge that was immediately run on a Nanostring nCounter SPRINT Profiler to determine fluorescent counts for each target. Raw RCC files were loaded into nSolver v4.0, and background thresholding was performed using the default parameters. Samples that did not pass the nSolver internal quality checks were removed. Thresholded count matrices were exported from nSolver and analyzed with [limma v3.62.1](https://doi.org/10.1093/nar/gkv007) ([edgeR v4.4.1](https://doi.org/10.1093/bioinformatics/btp616)) in R v4.4.2​.

## Data availability
The raw RCC files are available via NCBI GEO (accession TBD). The unformatted output figures are in [`analysis`](analysis/). The code is in [`analysis.r`](analysis.r).
